src/Http/Routing/src/Builder/EndpointRouteBuilderExtensions.cs

```diff
@@ -491,11 +494,16 @@ public static RouteHandlerBuilder MapFallback(this IEndpointRouteBuilder endpoin
             DisplayName = pattern.RawText ?? pattern.DebuggerToString(),
         };
 
-        // REVIEW: Should we add an IActionMethodMetadata with just MethodInfo on it so we are
-        // explicit about the MethodInfo representing the "handler" and not the RequestDelegate?
+        //var attributes = handler.Method.GetCustomAttributes();
 
-        // Add MethodInfo as metadata to assist with OpenAPI generation for the endpoint.
-        builder.Metadata.Add(handler.Method);
+        //// This can be null if the delegate is a dynamic method or compiled from an expression tree
+        //if (attributes is not null)
```

Comment:
What is going on here?

Reply:
Just left over from when it was moved to RDF. Will delete.